package/CHANGELOG

```diff
@@ -91,6 +91,8 @@ Fixes
   * Fixed selections using operators backwards ('prop 10 > mass') and sensitivity
     about whitespace around these (PR #1156 Issue #1011 #1009)
   * Fixed PSA analysis is now using AlignTraj
+  * The namespace used for auxiliary data now raises an AttributeError instead
```

Comment:
Wasn't NameSpace new in 0.16? If so, we don't have to document any fixes to it, (because CHANGELOG is for release to relrease changes)